bin/docker-entrypoint

```diff
@@ -25,13 +25,22 @@ dev_scheduler() {
 worker() {
   echo "Starting RQ worker..."
 
-  exec /app/manage.py rq worker $QUEUES
+  start_worker
 }
 
 dev_worker() {
   echo "Starting dev RQ worker..."
 
-  exec watchmedo auto-restart --directory=./redash/ --pattern=*.py --recursive -- ./manage.py rq worker $QUEUES
+  start_worker "watchmedo auto-restart --directory=./redash/ --pattern=*.py --recursive -- "
+}
+
+start_worker() {
+  export EXECUTION_PREFIX=$1
+  export WORKERS_COUNT=${WORKERS_COUNT:-2}
+  export QUEUES=${QUEUES:-}
+
+  touch /tmp/worker.log && tail -f /tmp/worker.log &
```

Comment:
Is there a reason you didn't configure supervisord to write to stdout directly?

Reply:
That's even simpler :) didn't think of that! 1fb1a7e handles this.